.circleci/config.yml

```diff
@@ -21,43 +24,61 @@ workflows:
               only: /^v.+\..+\..+/
   tests:
     jobs:
-      - test_installation
+      - test-installation
 
 
 jobs:
   lint:
     docker:
-      - image: circleci/python:3.6
+      - image: cimg/python:3.6
```

Comment:
I'm glad you asked! So, since dh-virtualenv really does just use virtualenv technology, we're not shipping our own Python binary. Instead, the virtualenv has a symlink to the system Python installation similar to how it would be set up on a dev machine. So, effectively, we're still stuck with whatever Python version is being used by the oldest distribution we support.

Reply:
I don't understand... If it relys on the system python, how can it work on Ubuntu 20, which doesn't carry python3.6

Reply:
dh-virtualenv seems to set up a symlink to "python3", whatever that ends up being in the system. In Ubuntu 20, that happens to be Python 3.8 and everything still works fine as far as I can tell. Like I mentioned above, though, I need to do more extensive testing on 20.04.

Reply:
Currently the CI tests run on 20.04, so I'd assume at least the installation works ;)